config/jobs/kubernetes-sigs/sig-windows/sig-windows-config.yaml

```diff
@@ -181,7 +181,7 @@ periodics:
     preset-dind-enabled: "true"
```

Comment:
just a note, the 1.14 bits above will probably have to be updated to 1.15 as well.